keel-core/src/test/kotlin/com/netflix/spinnaker/keel/services/ApplicationServiceTests.kt

```diff
@@ -367,6 +376,49 @@ class ApplicationServiceTests : JUnit5Minutests {
           that(states.first().type).isEqualTo("manual-judgement")
         }
       }
+
```

Comment:
these all just passthrough to the repository events, with no logic in the `ApplicationService`. Could you remove these tests? They're already tested at the repository level.

Reply:
I'll rewrite the tests to check that the repository is being called to persist the events.

Reply:
Is that really necessary to check, though? It's the only thing those methods are doing. I have 100% confidence that the repository is being called.

Reply:
Trying to stick to a pattern. I see @robfletcher used this approach extensively in the big test refactor in #1082. You have 100% confidence at the time you wrote the code, but one of the points of tests is to catch regressions that may be introduced by others who come later without prior context, and change that behavior inadvertently.